application/api/answer/routes.py

```diff
@@ -168,53 +169,53 @@ def complete_stream(question, retriever, conversation_id):
 
 @answer.route("/stream", methods=["POST"])
 def stream():
-    data = request.get_json()
-    # get parameter from url question
-    question = data["question"]
-    if "history" not in data:
-        history = []
-    else:
-        history = data["history"]
-        history = json.loads(history)
-    if "conversation_id" not in data:
-        conversation_id = None
-    else:
-        conversation_id = data["conversation_id"]
-    if 'prompt_id' in data:
-        prompt_id = data["prompt_id"]
-    else:
-        prompt_id = 'default'
-    if 'selectedDocs' in data and data['selectedDocs'] is None:
-        chunks = 0
-    elif 'chunks' in data:
-        chunks = int(data["chunks"])
-    else:
-        chunks = 2
-    
-    prompt = get_prompt(prompt_id)
-
-    # check if active_docs is set
-
-    if "api_key" in data:
-        data_key = get_data_from_api_key(data["api_key"])
-        source = {"active_docs": data_key["source"]}
-    elif "active_docs" in data:
-        source = {"active_docs": data["active_docs"]}
-    else:
-        source = {}
+    try:
+        data = request.get_json()
+        # get parameter from url question
+        question = data["question"]
+        if "history" not in data:
+            history = []
+        else:
+            history = data["history"]
+            history = json.loads(history)
+        if "conversation_id" not in data:
+            conversation_id = None
+        else:
+            conversation_id = data["conversation_id"]
+        if 'prompt_id' in data:
+            prompt_id = data["prompt_id"]
+        else:
+            prompt_id = 'default'
+        if 'selectedDocs' in data and data['selectedDocs'] is None:
+            chunks = 0
+        elif 'chunks' in data:
+            chunks = int(data["chunks"])
+        else:
+            chunks = 2
 
-    if source["active_docs"].split("/")[0] == "default" or source["active_docs"].split("/")[0] == "local":
-        retriever_name = "classic"
-    else:
-        retriever_name = source['active_docs']
+        # check if active_docs is set
 
-    retriever = RetrieverCreator.create_retriever(retriever_name, question=question, 
```

Comment:
I think during rebase you have removed some recent changes.
We should be using retrievers now for generating answers